cypress/component/annot-continuous-values.cy.js

```diff
@@ -42,8 +42,8 @@ describe("Continuous values component", () => {
                 },
                 getTransformOptions: () => (p_activeCategory) => {
 
-                    // return ["float", "bounded", "euro", "range", "int", "string", "isoyear"];
-                    return ["", "float", "bounded", "euro", "range", "int", "string"];
+                    // return ["float", "bounded", "euro", "int", "isoyear"];
```

Comment:
I know this was already here, but would you mind finding out why `isoyear` is commented out and then adding the info via a `TODO` flag to the comment so we know when to add it back (and remove this comment)

Reply:
Found the reason in the description of #368 PR
> `isoyear` transformation is currently being left (commented) out so it can be more fully addressed in a more robust way in its own new issue.

Couldn't find an issue to keep track of it so made #402